### PR TITLE
Multiqc for bp project

### PR DESCRIPTION
**Pull request by matrulda**
A simple wrapper for the multiqc command in INS-22349.
Usage: `bash /proj/ngi2016001/private/scripts/run_multiqc_bp_qc.sh /proj/ngi2016001/nobackup/NGI/ANALYSIS/<project>`

The command is run on a compute node.

### CODE REVIEW

File: run_multiqc_bp_qc.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -l
+
+#
+# A simple wrapper for the multiqc command in INS-22349
+# Usage: bash /proj/ngi2016001/private/scripts/run_multiqc_bp_qc.sh /proj/ngi2016001/nobackup/NGI/ANALYSIS/<project>
+#
+
```
Comment (b97pla) [inline run_multiqc_bp_qc.sh]: how about making this a sbatch script instead by adding the slurm parameters like this:

```suggestion
#SBATCH -A ngi2016001
#SBATCH -n 8
#SBATCH -t 05:00:00
#SBATCH -J run_multiqc_bp_qc
#SBATCH -o run_multiqc_bp_qc.%j.out
```

File: run_multiqc_bp_qc.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+PROJECT_PATH=$1
+
+sbatch -A ngi2016001 -n 8 -t 05:00:00 --wrap "multiqc --template default --config /proj/ngi2016001/private/conf/multiqc_config.yaml --title $(basename $PROJECT_PATH) --filename $(basename $PROJECT_PATH)_$(date +%y%m%d) --outdir multiqc_ngi --data-format json --zip-data-dir --no-push $PROJECT_PATH"
```
Comment (b97pla) [inline run_multiqc_bp_qc.sh]: ... and then just write the rest of the script as a regular bash script:

```suggestion
multiqc --template default --config /proj/ngi2016001/private/conf/multiqc_config.yaml --title "$(basename $PROJECT_PATH)" --filename "$(basename $PROJECT_PATH)_$(date +%y%m%d)" --outdir multiqc_ngi --data-format json --zip-data-dir --no-push "$PROJECT_PATH"
```

**Review by b97pla — COMMENTED** [review]
Nice, this looks correct to me! 

However, I would vote for making the script a sbatch-formatted script directly (see comments). This would have the benefit of being able to choose whether to submit the job to slurm or run locally, as well as being able to override slurm parameters from the command line.  

Let me know what you think!

**Comment by matrulda** [comment]
@b97pla Oh, did not know you could pass arguments to a sbatch script! Super useful. Adding your changes.

**Comment by matrulda** [comment]
@b97pla Feel free to merge if you think it looks OK.

**Review by b97pla — APPROVED** [review]
:+1: excellent, looks great!

**Pull request merged**